api/src/main/java/aero/minova/cas/api/restapi/ClientRestAPI.java

```diff
@@ -191,7 +191,8 @@ public ResponseEntity<byte[]> sendGetZipRequest(String path) {
 	public HttpStatus sendUploadLogRequest(byte[] log) {
 		HttpEntity<byte[]> request = new HttpEntity<>(log, createHeaders(username, password));
 		ResponseEntity<Void> response = restTemplate.exchange(url + "/upload/logs", HttpMethod.POST, request, Void.class);
-		return response.getStatusCode();
+
+		return HttpStatus.valueOf(response.getStatusCode().value());
```

Comment:
Welchen Zweck hat diese Änderung?

Reply:
Das Objekt vom Typ `org.springframework.http.HttpStatusCode` in `org.springframework.http.HttpStatus` umzuwandeln. Elegantere Lösungen sind jederzeit willkommen.

Reply:
Da hatte ich mich beim Typ verlesen.